### PR TITLE
Add a page for Staff roles and people

### PR DESCRIPTION
**Pull request by bvhme**
This page can serve to explain how the staff of the Foundation for Public Code is organized.
It includes the hierarchy as well as an overview of who is working in what position.

### CODE REVIEW

**Comment by nebster** [comment]
This document obviously has to include me in some way.

**Comment by bvhme** [comment]
I've added both @nebster and Floris by adding 'Contractors' to the page.

**Comment by bvhme** [comment]
For permalinking reasons I think that mixing 'staff' and 'contractors' might be a difficult issue.

I'm now thinking that leaving people off the staff page and adding a separate section for this might be a good idea, so that we can leave the description of the staff structure in place whilst also mentioning everyone.

File: organization/staff-and-contractors.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+type: Resource
+---
+
+# Staff and contractors
+
+The staff of the Foundation for Public Code vereniging are employees that execute the work agreed on by the [general assembly](../organization/governance-model.md#general-assembly) under direction of the [Board of directors](../organization/governance-model.md#board-of-directors).
+
+Some roles in our organization are fulfilled by contractors. these are mentioned with the legal entity they bill through for transparency.
```
Comment (ElenaFdR) [inline organization/staff-and-contractors.md]:
```suggestion
Some roles in our organization are fulfilled by contractors. For transparency, these are mentioned with the legal entity they bill through.
```

File: organization/staff-and-contractors.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+type: Resource
+---
+
+# Staff and contractors
+
+The staff of the Foundation for Public Code vereniging are employees that execute the work agreed on by the [general assembly](../organization/governance-model.md#general-assembly) under direction of the [Board of directors](../organization/governance-model.md#board-of-directors).
```
Comment (ElenaFdR) [inline organization/staff-and-contractors.md]:
```suggestion
The staff of the Foundation for Public Code Vereniging are employees that execute the work agreed on by the [general assembly](../organization/governance-model.md#general-assembly) under direction of the [board of directors](../organization/governance-model.md#board-of-directors).
```

File: organization/staff-and-contractors.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Some roles in our organization are fulfilled by contractors. these are mentioned with the legal entity they bill through for transparency.
+
+As we are a community leadership organization that mostly helps others we aim for minimal hierarchy and overhead and maximal autonomy for the specialists who further [our mission](mission.md).
```
Comment (ElenaFdR) [inline organization/staff-and-contractors.md]:
```suggestion
As we are a community leadership organization that mostly helps others, we aim for minimal hierarchy and overhead and maximal autonomy for the specialists who further [our mission](mission.md).
```

File: organization/staff-and-contractors.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+### President
+
+The president is the main spokesperson for the organization. The president represents the organization in order to raise funds, grow membership and grow awareness of our work.
+
+Roles:
+
+* President:
+  Ben Cerveny
+  (contractor Multiple Ventures BV)
+  who also serves as Chairman of the Board of Directors
```
Comment (ElenaFdR) [inline organization/staff-and-contractors.md]:
```suggestion
  Ben also serves as Chairman of the Board of Directors
```

File: organization/staff-and-contractors.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+* Hiring and HR
+* Value tracking and quantification
+* Procurement, office management and contract management
+* Finances, budget and resource management
+
+Roles:
+
+* [Chief](#chief) executive:
+  Boris van Hoytema
+  who also serves as Secretary of the Board of Directors
```
Comment (ElenaFdR) [inline organization/staff-and-contractors.md]:
```suggestion
  Ben also serves as Secretary of the Board of Directors.
```

**Pull request merged**